content/en/integrations/guide/aws-manual-setup.md

```diff
@@ -102,7 +102,7 @@ Ensure to leave **Require MFA** disabled.
 
 ### AWS
 
-1. In your AWS console, create an IAM user to be used by the Datadog integration with the [necessary permissions](#aws-integration-iam-policy).
+1. In your AWS console, create an IAM user to be used by the Datadog integration with the necessary permissions. Scroll to the end of this document for the full list.
```

Comment:
Hey @jtappa thanks for the review and suggestion, I agree that the current update is not ideal. As a workaround I've tried adding a `Permissions` heading and linking to that, but this still doesn't seem to work as expected (but let me know if I've set it up incorrectly). It does seem to work when opening the link in a new tab -- do you think it might be feasible to substitute an HTML link for the markdown, so that we can set the target to open in a new tab?